iOS/UZTickets/src/scan/UZTScanViewController.m

```diff
@@ -0,0 +1,54 @@
+//
+//  UZTScanViewController.m
+//  UZTickets
+//
+//  Created by nekoi on 6/1/15.
+//  Copyright (c) 2015 DAloG ltd. All rights reserved.
+//
+
+#import "UZTScanViewController.h"
+
+#import "UZTScanViewModel.h"
+#import "UZTScanSession.h"
+
+@interface UZTScanViewController ()
+
+@property UZTScanSession* scanningSession;
+
+@end
+
+@implementation UZTScanViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    self.scanningSession = [UZTScanSession new];
+    [self.view.layer insertSublayer:self.scanningSession.layer atIndex:0];
```

Comment:
I still recon that managing layers separately from view is more OS X way of working with display tree.